Insignia, Radios & Default Units Update

calumbrookes opened the pull request:
Gibbs has written us many functions. This is a holding PR while I check & test the dev branch.

Poulern committed:
Trimmed Folders, added group join

Still have a lot to go, but I'd figure I would share in the progress for now. I don't think I've removed anything we haven't used in over 2 years. Might cut down a megabyte or so.

dgibso29 committed:
Replaced existing insignia with Gibbs Insignia

Updated description.ext & assignInsignia.sqf to be compatible.

dgibso29 committed:
Simplifed Role insignia Assignment

Also fixed a typo.

dgibso29 committed:
Removed extra spaces

dgibso29 committed:
Update mission.sqm

calumbrookes committed:
Merge pull request #54 from dgibso29/Gibbs_Insignia

Replaced existing insignia with Gibbs Insignia

dgibso29 committed:
Removed GMG, HMG, COY

- Updated AAA to SAM, MOR to MTR
- Reverted ENG_SL & LOG_SL to DC gearscript entries

dgibso29 committed:
Properly set ENG_SL & LOG_SL to dc instead og eng/log

dgibso29 committed:
Added gearscript entries for Surgeon

- Gave all units 1 500ml Blood Bag
- Gave medics the Medical Bag

dgibso29 committed:
Added support for new Insignia & updated Platoon Setup to correspond.

dgibso29 committed:
Updated SmoothMarkers to include all current markers

dgibso29 committed:
Added new markers

AT, HAT, Heavy Weapons (IE MMG/HMG)

dgibso29 committed:
Fixed typo in ENG/LOG Group Names preventing Group recognition

dgibso29 committed:
Updated platoonSetup to support new SmoothMarkers

Poulern committed:
Removed more folders, removed deprecated folders

incl acre2 to its own folder
Lots of F3 stuff thats long been handled by ace, ca folder, cba or new arma 3 engine functionality.

Poulern committed:
Merge branch 'dev' of https://github.com/CombinedArmsGaming/F3_CA into dev

calumbrookes committed:
Merge pull request #56 from dgibso29/Updated_Default_Units

Update Default Units

calumbrookes committed:
Merge pull request #57 from dgibso29/Gibbs_Insignia

Update for Gibbs Insignia v1.1

calumbrookes committed:
Merge pull request #58 from dgibso29/UpdateSmoothMarkers

Update smooth markers

Poulern committed:
Fix radios not working correctly after folder change

dgibso29 committed:
Added SF & EOD Markers

Updated platoon setup to match.

calumbrookes committed:
Merge pull request #60 from dgibso29/Add-Additional-Markers

Add additional Smooth Markers

Poulern commented:
@dgibso29 
Advise changing group badges from group variable to groupid. will be more persistent with the rest of the framework, which uses groupid to identify units. It also doesn't require a check for side or faction saving quite a few lines AND if a unit merges into another it will more seamlessly reapply it on respawn. We could also make multiple groupids/callsigns correspond to one insignia like insignia_GI_ASL would apply to both groupids "ASL" and "ALPHA". (Though it would be cool to have both a "insignia_GI_ASL" and "insignia_GI_ALPHA" as separate as we do run single group squads in smaller ops. And while I am at it just a plain CO insignia would be great too!).

I am not sure what all of the unit insignia classes mean (eg what is a "pj"?), but its a lot more options with limited space concerns which is great!

Oh and don't forget we can type in the unit description: Alpha Squad Lead@ALPHA to get the slotting changed. might be worth adding on mission.sqm revision.

dgibso29 committed:
Update f_assignInsignia.sqf

dgibso29 committed:
Added XO & CO roles

calumbrookes committed:
Merge pull request #61 from dgibso29/Add-Role-Variable-to-AssignInsignia

Update f_assignInsignia.sqf

calumbrookes committed:
Tidying up misison.sqm

Making some editorial changes post gibbs' efforts over the last few days. This edit still requires the init edit for the CO & XO units to assign them the according XO and CO badges.

calumbrookes committed:
Edited LR Radio channels and added CO & XO patches.
